Makefile

```diff
@@ -39,7 +39,7 @@ MEMORY_REQUESTS ?= 200Mi
 MEMORY_LIMITS ?= 1639Mi
 GOOS ?= linux
 
-build_cmd = GO111MODULE=on mkdir -p _output && GOOS=$(GOOS) go build -o _output/$(1) ./cmd/$(1)
+build_cmd = GO111MODULE=on CGO_ENABLED=0 mkdir -p _output && GOOS=$(GOOS) go build -o _output/$(1) ./cmd/$(1)
```

Comment:
This depends a bit on the build env, but on my local machine they're still dynamically linked. To link statically I need to
```suggestion
build_cmd = GO111MODULE=on CGO_ENABLED=0 mkdir -p _output && GOOS=$(GOOS) go build -ldflags '-s -w -linkmode external -extldflags "-static"' -o _output/$(1) ./cmd/$(1)
```

Reply:
+1
One some machines, just disabling CGO won't statically build, the linker needs to be informed too.

Reply:
I also guess we have to put the `CGO_ENABLED=0` to the `go build` command:

```suggestion
build_cmd = GO111MODULE=on mkdir -p _output && CGO_ENABLED=0 GOOS=$(GOOS) go build -o _output/$(1) ./cmd/$(1)
```

Reply:
Done.